.github/workflows/build_and_test.yml

```diff
@@ -129,11 +135,21 @@ jobs:
         python3.8 -m pip install numpy 'pyarrow<3.0.0' pandas scipy xmlrunner
         python3.8 -m pip list
     # Run the tests.
+    - name: Install Oracle docker image
+      if: (contains(matrix.modules, 'docker-integration-tests'))
+      run: |
+        pushd /tmp
+        git clone https://github.com/oracle/docker-images.git
```

Comment:
Shall we use `SHA`? Otherwise, this will be another flaky test because the branch is out of our control.

Reply:
Yeah, It's better. Thanks.